.releaserc.yaml

```diff
@@ -1,6 +1,7 @@
 branches: 
```

Comment:
I missed the earlier PRs, but can you explain to me what this file is doing. I don't entirely understand

Reply:
It's the configuration file for [semantic-release](https://github.com/semantic-release/semantic-release).  It's just being set up with some basic config, plugins, and a preset that understands conventional commits.